addons/payment_stripe/models/payment.py

```diff
@@ -74,8 +74,15 @@ def _add_available_payment_method_types(self, stripe_session_data, tx_values):
             PMT('p24', ['pl'], ['eur', 'pln'], 'punctual'),
         ]
 
+        existing_icons = [i.name.lower() for i in self.env['payment.icon'].search([])]
```

Comment:
This way it's clearer that both lines refer to the same objects:
```suggestion
        existing_icons = [icon.name.lower() for icon in self.env['payment.icon'].search([])]
```

Reply:
done